frontend/public/components/resource-quota.jsx

```diff
@@ -29,7 +33,54 @@ const Details = ({obj: rq}) => <React.Fragment>
 </React.Fragment>;
 
 export const ResourceQuotasList = props => <List {...props} Header={Header} Row={Row} />;
-export const ResourceQuotasPage = props => <ListPage {...props} ListComponent={ResourceQuotasList} kind={kind} canCreate={true} />;
+
+export const quotaType = quota => {
+  if (!quota) {
+    return undefined;
+  }
+  return quota.metadata.namespace ? 'namespace' : 'cluster';
+};
+
+// Split each resource quota into one row per subject
+export const flatten = resources => _.flatMap(resources, resource => {
+  const rows = [];
+
+  _.each(resource.data, quota => {
+    if (!quota) {
+      return undefined;
+    }
+    rows.push(quota);
+  });
+
+  return rows;
```

Comment:
Assuming resource.data is an array, I think this can be... (untested)

```js
export const flatten = resources => _.flatMap(resources, resource => _.compact(resource.data));
```